pkg/dataloader/prowloader/prow.go

```diff
@@ -515,12 +516,9 @@ func (pl *ProwLoader) prowJobToJobRun(ctx context.Context, pj *prow.ProwJob, rel
 		return err
 	}
 
-	parts := strings.Split(pjURL.Path, pl.bktName)
```

Comment:
This code was splitting on the bucket name, but at the moment the bucket name is test-platform-results but the path still has origin-ci-test.  I moved to a regexp instead.